hack/run-e2e.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TIDB_OPERATOR_IMAGE=${TIDB_OPERATOR_IMAGE:-localhost:5000/pingcap/tidb-operator:latest}
+E2E_IMAGE=${E2E_IMAGE:-localhost:5000/pingcap/tidb-operator-e2e:latest}
+TEST_APISERVER_IMAGE=${TEST_APISERVER_IMAGE:-localhost:5000/pingcap/test-apiserver:latest}
+KUBECONFIG=${KUBECONFIG:-}
+
+if [ -z "$KUBECONFIG" ]; then
+    echo "error: KUBECONFIG is required"
+    exit 1
+fi
+
+echo "TIDB_OPERATOR_IMAGE: $TIDB_OPERATOR_IMAGE"
+echo "E2E_IMAGE: $E2E_IMAGE"
+echo "TEST_APISERVER_IMAGE: $TEST_APISERVER_IMAGE"
+echo "KUBECONFIG: $KUBECONFIG"
+
+NS=tidb-operator-e2e
+
+# copy and modify to avoid local changes
+# TODO find a better way
```

Comment:
Is it possible to use `cat tests/manifests/e2e/e2e.yaml | sed 'xxx' | kubectl apply -f -`?

Reply:
done